master/internal/task/allocation.go

```diff
@@ -184,6 +187,14 @@ func (a *Allocation) Receive(ctx *actor.Context) error {
 		if err := a.RequestResources(ctx); err != nil {
 			a.Error(ctx, err)
 		}
+
+	case IsAllocationInProgressOfRestore:
+		if a.req.Restore && !a.restored {
+			ctx.Respond(ErrAllocationStillRestoring{})
+		} else {
+			ctx.Respond(true)
```

Comment:
could just response nil too. true is kinda a confusing response given the message name

Reply:
Responding nil seems to make the ```apiServer.ask``` convert it to a 404 error. 

Changed the message name to ```AllocationNotRestoring``` to avoid sending true when the name indicates the opposite.

Reply:
Why not is allocation restoring? and return false/true?

Reply:
That is a lot better, changed to that